vendor/github.com/sylabs/sif/pkg/sif/fmt.go

```diff
@@ -89,6 +89,8 @@ func fstypeStr(ftype Fstype) string {
 		return "Archive"
 	case FsRaw:
 		return "Raw"
+	case FsEncrypt:
```

Comment:
need to revert this change

Reply:
Yes. I will revert the vendor changes once the PR in SIF repo is merged.